packages/edit-post/src/index.js

```diff
@@ -10,6 +10,7 @@ import { render, unmountComponentAtNode } from '@wordpress/element';
 import { dispatch, select } from '@wordpress/data';
 import { addFilter } from '@wordpress/hooks';
 import { store as preferencesStore } from '@wordpress/preferences';
+import '@wordpress/database-persistence-layer';
```

Comment:
Adding this was the only way I found to make the package available as `wp.databasePersistenceLayer` for the inline script.

There's probably another way, it would be great to hear if there is.